hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1511,6 +1511,31 @@
       milestone: V2
       tests: []
     }
+    {
```

Comment:
Sorry, one quick comment - I have duplicated the tests for main and ret sram above - could you consolidate them instead, and indicate in the descr to target both flavors.

Reply:
sounds good! will update this one.